Source/Core/AudioCommon/WaveFile.h

```diff
@@ -19,14 +19,18 @@
 
 #include "Common/CommonTypes.h"
 #include "Common/File.h"
-#include "Common/NonCopyable.h"
 
-class WaveFileWriter : NonCopyable
+class WaveFileWriter
 {
 public:
   WaveFileWriter();
   ~WaveFileWriter();
 
+  WaveFileWriter(const WaveFileWriter&) = delete;
+  WaveFileWriter& operator=(const WaveFileWriter&) = delete;
+  WaveFileWriter(WaveFileWriter&&) = delete;
```

Comment:
Ah, OK. IMO, `Wiimote` and `NativeVertexFormat` should still be left as movable, since the destructor doesn't have any immediate side-effects.